master/internal/resourcemanagers/setup.go

```diff
@@ -97,7 +97,7 @@ func setupAgentResourceManager(
 	)
 	system.Ask(ref, actor.Ping{}).Get()
 
-	agent.Initialize(system, echo, opts)
+	agent.Initialize(system, echo, config.ResourceManager.AgentRM, opts)
```

Comment:
should this be `agent.Initialize(system, echo, opts, config.ResourceManager.AgentRM)` ? Looks like the function def has the config and opts reversed.

Reply:
Welp, that's what I get for deciding to move things around right at the end of the day.